styles/nodejs/catppuccin.user.css

```diff
@@ -0,0 +1,230 @@
+        @overlay2: @catppuccin[@@lookup][@overlay2];
+        @overlay1: @catppuccin[@@lookup][@overlay1];
+        @overlay0: @catppuccin[@@lookup][@overlay0];
+        @surface2: @catppuccin[@@lookup][@surface2];
+        @surface1: @catppuccin[@@lookup][@surface1];
+        @surface0: @catppuccin[@@lookup][@surface0];
+        @base: @catppuccin[@@lookup][@base];
+        @mantle: @catppuccin[@@lookup][@mantle];
+        @crust: @catppuccin[@@lookup][@crust];
+        @accent-color: @catppuccin[@@lookup][@@accent];
+
+        color-scheme: if(@lookup =latte, light, dark);
+
+        ::selection {
+            background-color: fade(@accent-color, 30%);
+        }
+
+        input,
+        textarea {
+            &::placeholder {
+                color: @subtext0 !important;
+            }
+        }
+
+        --black: @crust;
+        --black1: @mantle;
+        --black2: @overlay2;
+        --black3: @overlay0;
+        --blue1: @blue;
+        --white: @base;
+        --white-smoke: @surface0;
+        --grey-smoke: @surface1;
+        --red1: @red;
+        --red2: @red;
+        --red3: @peach;
+        --red4: @flamingo;
+        --green1: @green;
+        --green2: @teal;
+        --green3: @green;
+        --green4: @surface2;
+        --green5: @overlay1;
+        --gray1: @subtext1;
+        --gray2: @overlay0;
+        --gray3: @surface2;
+        --gray4: @overlay2;
+        --gray5: @subtext0;
+        --gray6: @text;
+        --gray7: @surface1;
+        --grey8: @mantle;
+        /*         --background-color-api-stability-link: rgba(239, 241, 245, 0.4); */
+        --background-color-api-stability-link: @blue;
+        --background-color-highlight: @surface0;
+        --color-brand-primary: @text;
+        --color-brand-secondary: @green;
+        --color-critical: @red;
+        --color-fill-app: @base;
+        --color-fill-side-nav: @text;
+        --color-links: @accent-color;
+        --color-text-mark: @subtext1;
+        --color-text-nav: @surface2;
+        --color-text-primary: @text;
+        --color-text-secondary: @teal;
+
+        code {
+            color: @text;
+            background-color: @mantle;
+        }
+
+
+        a code,
+        a {
+            color: @accent-color;
+        }
+
+        a:hover,
+        a code:hover {
+            background-color: @surface0;
+        }
+
+        pre {
+            background-color: @mantle;
+        }
+
+        #column2 li,
+        #intro {
+            background-color: @mantle !important;
+        }
+
+        #intro a {
+            color: @text;
+        }
+
+        #column2 li:hover {
+            background-color: lighten(@mantle, 10%) !important;
+        }
+
+        #column2 hr {
+            border-color: @surface0 !important;
+        }
+
+        #column2 li a.active {
+            color: lighten(@text, 10%)
+        }
+
+        #column2 li a {
+            color: @text;
+            text-decoration: none;
+        }
```

Comment:
Try to use nesting when possible: https://lesscss.org/features/#parent-selectors-feature.